src/index.ts

```diff
@@ -162,9 +172,9 @@ class Paragraph {
    *
    * @param {PasteEvent} event - event with pasted data
    */
-  onPaste(event) {
+  public onPaste(event: HTMLPasteEvent) {
```

Comment:
it is not an HTMLPasteEvent, we pass our own custom Event here. I guess Editor.js should export its type.

See https://editorjs.io/tools-api#onpaste